### ISSUE TITLE
eBPF Performance Test

### ISSUE THREAD

**Issue by danehans** [solo-io/ambient-performance#10]
Create a test that uses eBPF instead of iptables.

**Comment by bleggett** [comment]
I'm not convinced that there will ever be much actual diff. between eBPF and iptables perf - or at least it will be tiny.

The perf diff that concerns me is between userspace proxies - ztunnel vs envoy - we know already that ztunnel HBONE across nodes is slower than Envoy, correct?

Any eBPF perf differences will be entirely subsumed by that.

**Comment by howardjohn** [comment]
I agree. It is still good to benchmark it though. A _correct_ implementation of both is likely similar, but a benchmark could expose us doing something weird

**Comment by danehans** [comment]
IMO having the performance data will help even if the results are negligible.
